### DISCUSSION TITLE
药王慈怀！建木生发！莳者一心！同登极乐！

### DISCUSSION THREAD

**Discussion by ehan0828** [share121/inter-knot#3343] [其他]
如是我闻: 
一时千手慈怀药王游化诸界，与[莳者]九万九千九百九十九人俱，住罗喉下，而为说法。
尔时倏忽令使问千手慈怀药王言:【众生有疾，万类皆苦。一切有情众生，缚于生老病苦，何谓苦本？】 
千手慈怀药王言: 【生老病苦，皆归于死。生者之苦，苦于有涯。老者之苦，苦于必至。病者之苦，苦于
近死。是故生老病苦，谓有死。从有死，则为苦本。】 
倏忽令使合掌言:【一切有情众生，有灭尽日，有轮回日，何断生老病苦？】 
千手慈怀药王告倏忽令使言:【是彼众生，囿于形骸，如囚入笼，受三苦刑。吾今植永寿神木，令彼众生
解脱，生无涯，老不至，死回生，断离烦恼。】 
尔时倏忽令使作是念言:【药王慈怀，建木生发。莳者一心，同登极乐。】 
【药王慈怀，建木生发。莳者一心，同登极乐。】 
尔时莳者九万九千九百九十九人俱言:【药王慈怀，建木生发。莳者一心，同登极乐。 】
说是语时，莳者九万九千九百九十九人俱，皆断生老病苦。
如果有想要加入兄弟姐妹们的求药使，请联系吾。现在加入药王秘传，即可免费领取一本《千手慈怀药王救世品》！
联系电话：11451141919810
![Uploading e7408d5f555ede3d7ebc817f821493a.png…]()

**Comment by ehan0828** [comment]
有一处编写错误：
不应该是：
苦于
近死。是故生老病苦，谓有死。从有死，则为苦本。】
而是：
苦于近死。是故生老病苦，谓有死。从有死，则为苦本。】
实在抱歉。

**Comment by lllliufeng** [comment]
我去，联觉信标连到绳网了
